src/harmonic.cpp

```diff
@@ -1,8 +1,9 @@
+#include <common.h>
 #include <npe.h>
 #include <typedefs.h>
 #include <igl/harmonic.h>
 
-//TODO: is is only dense_int because min_quad_with_fixed_data hardcodes int, fixme
+//TODO: is is only dense_int because min_quad_with_fixed_data hardcodes int, FIXME
```

Comment:
Is this comment still true?

Reply:
It seems min_quad_with_fixed has been templated now, I will try to fix this later